diff -urN aaa/glade-2.0.1/glade-2.desktop.in glade-2.0.1/glade-2.desktop.in
--- aaa/glade-2.0.1/glade-2.desktop.in	2003-10-30 19:59:45.000000000 +0100
+++ glade-2.0.1/glade-2.desktop.in	2003-12-12 18:10:24.000000000 +0100
@@ -2,10 +2,11 @@
 _Name=Glade Interface Designer
 _Comment=Design user interfaces
 Exec=glade-2 %F
-Terminal=0
+Terminal=false
 Type=Application
 Icon=glade-2.png
 Categories=Application;Development;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=glade
 X-GNOME-Bugzilla-Component=general
+Encoding=UTF-8
